READING_LISTS/ReproducibilityAndOpenScience.md

```diff
@@ -0,0 +1,42 @@
+This list is adapted from Lucia Magis Weinberg's [list of resources for getting started with open science and reproducibility](https://luciamagisweinberg.wordpress.com/2017/03/23/reproducibilityopen-science-resources/)
+
+### Some overviews and timelines of the "Reproducibility Crisis"
```

Comment:
I don't think we need "Some" in the headings - we know this isn't everything and I think shorter titles will make the reading list easier to scan over.